### ISSUE TITLE
[CV] Request new entry: total ion current max (until RT)

### ISSUE THREAD

**Issue by julianu** [HUPO-PSI/mzQC#248]
### What is the QC term's name?

total ion current max (until RT)

### Briefly describe the QC term.

This metric should capture the maximum value of a run's total ion current (TIC) (until a given retention time)

Like in #247 and #246 this could either be one or two metrics (one global, one until a given retention time)

This could maybe be modeled using a tuple or table specifying the RT and the maximum TIC until this RT (while maybe RT=-1 means globally)

### What is the QC term's unit?

intensity / arbitrary and time (in seconds)

### Value type

MS:4000004 ! n-tuple

### Describe any additional information.

- this term is ID-free and can easily be calculated using all spectra (MS1 and MS2)
- it is applicable to a single run
- the value should be (relatively) constant for same / similar samples

**Comment by bittremieux** [comment]
We have related terms `MS:4000029 ! area under TIC in MS1` and `MS:4000030 ! area under TIC in MS2`, this would be the sum of those two? We also have `MS:4000054 ! TIC quarters RT fraction`.

**Comment by julianu** [comment]
No, this would not be the area under the TIC, but the maximum of all the single intensities in the TIC (up to RT, or as suggested in #247 ) in an RT range.

**Comment by julianu** [comment]
Terms will be added to #263
